### DISCUSSION TITLE
#56 togETHer biweekly meeting

### DISCUSSION THREAD

**Discussion by 0xPabloLI** [tog-ETH-er/togETHer#41] [General]
Agenda:

-Community updates
-Fireside discussion

会议主题：togETHer
会议时间： 21:00-22:00
重复周期：每两周 (周一)

点击链接入会：
https://meet.google.com/jdg-wioj-rsj

**Comment by 0x-stan** [comment]
# Dapp-Learning

## repo update

11个PR #915 ～ #926

https://github.com/Dapp-Learning-DAO/Dapp-Learning/pulls

## sharing meeting

近两周举办了4场分享会

- 《How to choose data provider as web3 developer》 by Navy
  - https://www.youtube.com/watch?v=BOPjBMwFz_Y

- 《cross-chain technology》by Kevin
  - https://www.youtube.com/watch?v=REQ9KKYKcQQ
  
- 《Account Abstraction in Practice part1》by Arc
  -  https://www.youtube.com/watch?v=1m9QmyDCpFc
  - https://www.youtube.com/watch?v=RgLkyUGPufk

- 《MPC Wallet internals and practices》by ququzone
  - https://www.youtube.com/watch?v=Da9dhEK3vg0

## sharing meeting reward

- [DLIP-032 Reward the new contributors of the sharing meetup [1440 USDC]](https://github.com/Dapp-Learning-DAO/DLIP/issues/32)

**Comment by liushooter** [comment]
## Rebase

14期日报 1.23 ~2.5 https://rebase.network/posts/category/web3daily

**Comment by muziknozik** [comment]
### Plancker DAO
---

每天晚上19:30: Plancker英语共学 
🙋报名领读：
https://docs.google.com/spreadsheets/d/1YOwxQB3QJEsPRSw256_bzpcpqCCA75tM7SuYXlpnKs4/edit?usp=sharing
📒 活动材料：
https://plancker.notion.site/Plancker-43ad34a8cb8f47dba343b6f18599ae0a

![image](https://user-images.githubusercontent.com/2011052/216963866-ebc544c4-62e1-4403-aabe-3507ea9a93a7.png)

---

周五（2.3）：当优雅的NounsDAO遇上治理攻击 
👨 分享人：Eliade、屏风
⌚️ 分享时间：02/03/2022 16:00（UTC+8）
📒 活动材料：
https://plancker.notion.site/NounsDAO-2c46db0534b8470fa5dd480a607de4a1

---

周六（2.4）：研究周会#22 & 小V共读会#9 16:00（UTC+8）

💬 内容简介
本期泛读 The Infinite Machine-How an Army of Crypto-Hackers Is Building the Next Internet with Ethereum，让我们一起回顾以太坊创业史，《The Infinite Machine》记录了以太坊从创立（2014年）到ICO泡沫结束（2018-19年左右）的5年历程，近期还被改编成了电影🎬

💬 活动详情：
https://plancker.notion.site/22-V-9-56d4f51eaa8446359519827077d771c4

---
和 ECN 一起支持发起的 [Vitalik 中文博客同步](https://greenpillcn.notion.site/Vitalik-8092292f74064885930dc3e79397184f) 项目开始尝试实施。
和 ECN，官网翻译小组共同支持的 [以太坊翻译术语表](https://greenpillcn.notion.site/be0060a237cf4d61ae2740a5077479ef) 项目，正在酝酿，很快开始实施。

---

🔔 普朗克活动日历订阅
网页订阅： https://cutt.ly/aBYM4xq
本地订阅：https://cutt.ly/LBY1tNn

🏚️普朗克社区主页：
https://plancker.notion.site

关注 Plancker DAO 油管/B站，收获精彩完整的活动视频：
油管：
https://www.youtube.com/channel/UCatIOSch3OIVifZuK9jO7cg
Bilibili：
https://space.bilibili.com/1729402830

**Comment by Emilyxiannv** [comment]
### DAOSquare
努力推进孵化器产品Phoenix的开发中，产品最新进展：
https://app.musex.io/article/14/32

**Comment by Stamp9** [comment]
## 原语里弄

课题日常跟进。

**Comment by bambalados** [comment]
**Crypto Tech Night**

- CTN 20, 主题是Web3 in China

**Comment by 0x68616f79756e** [comment]
ETHPlanet

1.举办了以太坊Road map 学习讨论。
2.Academic Grants Round 2023 的讨论。

**Reply by 0x68616f79756e** [reply]
Academic Grants Round 2023：https://notes.ethereum.org/@djrtwo/academic-grants-wishlist-2023

**Comment by Riely007** [comment]
### 翻译小组

- 日常翻译进行中
